lib/rubocop/cop/style/yoda_condition.rb

```diff
@@ -20,7 +20,15 @@ module Style
       #   foo == 99
       #   foo == "bar"
       #   for <= 42
+      #
+      #   Supports EnforcedStyles:
```

Comment:
See the documentation for other cops with supported styles - there should be different examples for those.